.github/workflows/arm-ci.yml

```diff
@@ -21,5 +21,5 @@ jobs:
           cmake -S . -B build \
             -DCMAKE_TOOLCHAIN_FILE=${DEVKITPRO}/cmake/3DS.cmake
       - name: Build
-        run: make CXXFLAGS="-w -DHB_NO_MT"
+        run: make CXX_FLAGS="-w -DHB_NO_MT"
```

Comment:
Humm. Isn't `CXXFLAGS` correct?

Reply:
Or is cmake's different?

Reply:
Yeah this confused me too - CMake templates the Makefile with its own parameters. Weirdly setting these build flags at CMake configure time causes its compiler check to fail, which is why I'm doing it like this and not with `cmake --build`.